lib/oli_web/controllers/page_delivery_controller.ex

```diff
@@ -57,6 +57,40 @@ defmodule OliWeb.PageDeliveryController do
     end
   end
 
+  def exploration(conn, %{"section_slug" => section_slug}) do
+    user = conn.assigns.current_user
+    section = conn.assigns.section
+
+    if Sections.is_enrolled?(user.id, section_slug) do
```

Comment:
Should this work for admins as well? If so, wouldn't it fail because the admin is not enrolled to the section?

Reply:
mmm I understand. I could add one more test that contemplates the situation in which one enters as admin.